src/views/Settings.vue

```diff
@@ -1,238 +1,219 @@
 <template>
-  <v-container :style="slim && 'max-width: 80vw; width:500px;'" :class="{ 'pa-0': slim }">
-    <v-row dense>
-      <v-col v-if="!slim" cols="12">
-        <div class="text-h4 mb-2 ml-5 mt-2">
+  <v-container>
```

Comment:
some users actually do enjoy the Quick Settings (me for example who change clip languages a lot)

It might be good to still retain that, not sure what this new UI looks like tho i might check the PR out and see.